models/public/vehicle-license-plate-detection-barrier-0123/description/vehicle-license-plate-detection-barrier-0123.md

```diff
@@ -0,0 +1,54 @@
+# vehicle-license-plate-detection-barrier-0123
+
+## Use Case and High-level Description
+
+This is a MobileNetV2 + SSD-based vehicle and (Chinese) license plate detector for
+the "Barrier" use case.
+
+## Example
+![](./vehicle-license-plate-detection-barrier-0123.jpg)
+
+## Specification
+
+| Metric                          | Value                                      |
+|---------------------------------|--------------------------------------------|
+| Mean Average Precision (mAP)    | 99.52%                                     |
+| AP vehicles                     | 99.90%                                     |
+| AP plates                       | 99.13%                                     |
+| Car pose                        | Front facing cars                          |
+| Min plate width                 | 96 pixels                                  |
+| Max objects to detect           | 200                                        |
+| GFlops                          | 8.666                                      |
+| MParams                         | 0.547                                      |
+| Source framework                | TensorFlow*                                |
+
+Average Precision (AP) is defined as an area under the
+[precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve. Validation dataset is BIT-Vehicle.
+
+## Performance
+
+## Inputs
+
+1. name: "input" , shape: [1x3x256x256] - An input image in the format [BxCxHxW],
```

Comment:
for public models we provide 2 section inputs/outputs if model has difference in these layers. As I see this model has `RGB` input in source framework format

Reply:
Please, explain what you mean more specifically

Reply:
when user download public model, he or she makes deal with model in source framework format and after that convert it to IR, preferable way to made it to use converter, but it can be done also manually or even user can use it in source framework format (e.g. load via opencv or in tensorflow). I think we should warn about difference between source framework model and IR if it exists (I mean if model have mean/scale or require image in RGB, when these parameters are a part of IR)